feature-libs/cart/base/core/facade/multi-cart.service.spec.ts

```diff
@@ -232,15 +242,19 @@ describe('MultiCartService', () => {
         new CartActions.LoadCart({
           userId: 'userId',
           cartId: 'xxx',
+          extraData: {
+            active: true,
+          },
         })
       );
+
       service
-        .isStable('cartId')
+        .isStable('xxx')
```

Comment:
just a note for the review,

this test was broken since the creation of it because when it got to the selectors, it was using an initialLoaderState because the cartId was not the same as the one that was dispatched due to links below

- https://github.com/SAP/spartacus/blob/develop/projects/core/src/state/utils/entity-processes-loader/entity-processes-loader.selectors.ts#L30
- https://github.com/SAP/spartacus/blob/develop/projects/core/src/state/utils/entity-loader/entity-loader.selectors.ts#L15